ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/MGMT_GATEWAY_TESTS.yml

```diff
@@ -0,0 +1,39 @@
+---
+# tests for mgmt_gateway and ipv6_mgmt_gateway inside node_type (l2leaf)
+
+default_node_types:
+  - node_type: l2leaf
+    match_hostnames:
+      - "MGMT-GATEWAY-IN-NODE-GROUP-NODES"
+      - "MGMT-GATEWAY-IN-NODE-GROUP"
+      - "IPV6-MGMT-GATEWAY-IN-NODE-GROUP-NODES"
+      - "IPV6-MGMT-GATEWAY-IN-NODE-GROUP"
```

Comment:
I don't think you need to split ipv4 and ipv6 in separate hosts. Also we don't need to test node groups and nodes etc for every feature. We test that generally somewhere, so a single host will probably suffice for this feature.

Reply:
Updated!